webui/react/src/services/api.ts

```diff
@@ -149,7 +164,7 @@ export const killTask = async (task: AnyTask, cancelToken?: CancelToken): Promis
   });
 };
 
-export const login = generateApi<Credentials, void>(Config.login);
+export const login = generateApi<Credentials, {token: string}>(Config.login);
```

Comment:
**nit:** `{ token: string }` probably not enforced by lint because it's TS specific and not JS

Reply:
I probably should define a named type and use that.